requirements/requirements.in

```diff
@@ -1,59 +1,59 @@
-apache-libcloud==2.0.0
+apache-libcloud==2.2.1
 appdirs==1.4.2
-asgi-amqp==1.1.1
 asgiref==1.1.2
-azure==2.0.0rc6
+azure==3.0.0
 backports.ssl-match-hostname==3.5.0.1
-boto==2.46.1
-boto3==1.4.4
+boto==2.48.0
+boto3==1.6.2
 channels==1.1.8
 celery==3.1.25
-daphne==1.3.0
-Django==1.11.7
+daphne==1.3.0   # Last before changed to channels 2
+Django==1.11.10
 django-auth-ldap==1.2.8
 django-celery==3.2.2
-django-crum==0.7.1
-django-extensions==1.7.8
+django-crum==0.7.2
+django-extensions==2.0.0
 django-jsonfield==1.0.1
 django-oauth-toolkit==1.0.0
-django-polymorphic==1.3
+django-polymorphic==2.0.2
 django-pglocks==1.0.2
-django-radius==1.1.0
-django-solo==1.1.2
-django-split-settings==0.2.5
-django-taggit==0.22.1
-djangorestframework==3.7.3
+django-radius==1.3.2
+django-solo==1.1.3
+django-split-settings==0.3.0
+django-taggit==0.22.2
+djangorestframework==3.7.7
 djangorestframework-yaml==1.0.3
-gevent-websocket==0.9.5
-irc==15.1.1
+gevent-websocket==0.9.5   # pinned because of django channels
```

Comment:
@chrismeyersfsu can I start putting comments to the side like this? This is the manually curated list, but I feel like anyone coming by would have no idea why anything was what it was.

Reply:
I don't think this is needed by django channels?  This was our old websocket interface requirement

https://github.com/django/channels/blob/master/setup.py#L14-L27

Reply:
Let me ping @wwitzel3 about that.

But I'm happy to remove them and carry on. Some people have already asked me to verify that the UI websockets are working, so I'll do this anyway.

Reply:
The comment in the README is

> Packages gevent-websocket and twisted are put in requirements.in not because they are primary dependency of Tower, but because their versions needs to be freezed as dependencies of django channel. Please be mindful when doing dependency updates.

I don't know if this still applies for current dependencies.